src/WriteModels/OrderUpdate.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+declare(strict_types=1);
+
+namespace VaniloCloud\WriteModels;
+
+use VaniloCloud\Enums\FulfillmentStatus;
+use VaniloCloud\Enums\OrderStatus;
+
+class OrderUpdate extends BaseWriteModel
+{
+    public function setStatus(OrderStatus $status): self
+    {
+        $this->data['status'] = $status;
+
+        return $this;
+    }
+
+    public function setFulfillmentStatus(?FulfillmentStatus $fulfillment_status): self
+    {
+        $this->data['fulfillment_status'] = $fulfillment_status;
```

Comment:
Let's use camelCase variables whenever possible:

```suggestion
    public function setFulfillmentStatus(?FulfillmentStatus $fulfillmentStatus): self
    {
        $this->data['fulfillment_status'] = $fulfillmentStatus;
```

Reply:
Apply this to all method arguments please

Reply:
I agree; it makes more sense to be consistent across the whole codebase and not make exceptions like this.

Reply:
I updated the files related to this PR. I want to mention that the previous models were created this way; I will update them in a future PR.